CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.0.2
+* Enabled to sign PUT methods
```

Comment:
I'd write this
> Enabled to sign PUT methods

as

> Allow signing PUT requests

to remain consistent with the other changeling entries.